src/apps/rDSN.monitor/app_package/static/js/service_meta.js

```diff
@@ -59,26 +59,29 @@ var vm = new Vue({
                                 if(par.primary!='invalid address')
                                 {
                                     par.membership += 'P: ("' + par.primary + '"),\n ';
```

Comment:
I see, could you help suggesting another possible way to check? I cannot come up with one